### PR TITLE
BUG: area_mode="range" in viz._plot_psd is wrong.

### PR DESCRIPTION
**Pull request by jshanna100**
As pointed out by #6500, the changes done in #6451 broke average plotting when area_mode="range." This fixes that, and also sets linewidth=0.5, and passes along the line_alpha parameter to plot.

### CODE REVIEW

**Comment by larsoner** [comment]
@jshanna100 does this fix the titles? I don't see how it could from the code but maybe I'm missing it.

File: doc/whats_new.rst [hunk 1]
```diff
@@ -45,6 +45,8 @@ Changelog
 Bug
 ~~~
 
+- Fix ``area_mode='range'`` in :func:`mne.viz.utils._plot_psd` Also line width set back to 0.5, und line_alpha is passed through. by `Jeff Hanna`
```
Comment (larsoner) [inline doc/whats_new.rst]: Only reference public functions in release notes, so maybe
```
:meth:`mne.Epochs.plot_psd`
```
? Also und->and.

Finally, if all this PR does is restore previously working behavior within a given release cycle, you actually don't need to add to `whats_new.rst`. So this should only be here for example if it fixes a bug that existed with `Raw.plot_psd`.

Comment (jshanna100) [reply doc/whats_new.rst]: Yes, sorry, of course. This should now fix the titles.

**Comment by agramfort** [comment]
thx @jshanna100

**Pull request merged**